tensorrt_llm/_torch/models/modeling_nemotron_h.py

```diff
@@ -258,7 +260,7 @@ def load_weights(self, weights: Dict):
         config = self.model_config.pretrained_config
         tp_size = self.model_config.mapping.tp_size
         tp_rank = self.model_config.mapping.tp_rank
-        d_inner = config.hidden_size * config.expand
```

Comment:
Generally, why is the change from computing `d_inner` using `expand` to computing it by `mamba_head_dim*mamba_num_heads` necessary? Can you add an explanation in the PR description?

Reply:
this is what @JRD971000 mentioned about it:

"Previously, the number of mamba heads was affected by the hidden size of the model; mamba_num_heads = expand * hidden_size / mamba_head_dim. With this refactor, mamba_num_heads is set directly, and it is decoupled from the hidden size of the model."

Link [here](https://github.com/vllm-project/vllm/pull/19766#issuecomment-2982020537)

Reply:
Got it. Worth adding an explanation in PR description for easier tracking in the future